LICENSE.imagination

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015 Imagination Technologies Limited and/or its affiliated group companies
```

Comment:
Should this be in "firmware" folder, rather than here?
@chaitu-tk what is your opinion?